### ISSUE TITLE
Does the multicluster setup support  coordination between the controllers in the two clusters?

### ISSUE THREAD

**Issue by kapilagrawal95** [adavarski/kind-linkerd-multicluster#1]
I have a fundamental question. Doesthis multi cluster setup synchronizes the etcd of each cluster as well? For example, if I have a deployment which I would like to ensure are spread uniformly across both the clusters? How will the coordination between the controllers be in such a situation?

**Comment by drewwells** [comment]
no, that's not how linkerd multicluster works. There's references at the end of the readme to explain how multicluster works
